Assets/HoloToolkit/SpatialMapping/Scripts/DataEventArgs.cs

```diff
@@ -3,7 +3,7 @@
 
 using System;
 
-namespace HoloToolkit.Unity
+namespace HoloToolkit.Unity.SpatialMapping
```

Comment:
This (and PropertyChangedEventArgsEx.cs) look more generic than just for use in spatial mapping. Should they go in the common folder instead?

Reply:
We spoke offline. Decided on meh!